pkg/upgrade/upgrade.go

```diff
@@ -351,6 +351,46 @@ func UpdateFromLegacyVersion(cli client.Client, platform deploy.Platform, appNS
 	return nil
 }
 
+// CleanupDeprecatedFeatures deletes deprecated features by removing the corresponding FeatureTrackers
+func CleanupDeprecatedFeatures(ctx context.Context, cli client.Client, platform deploy.Platform) error {
+	if platform != deploy.ManagedRhods {
+		return nil
+	}
+
+	crd := &apiextv1.CustomResourceDefinition{}
+	if err := cli.Get(ctx, client.ObjectKey{Name: "featuretrackers.features.opendatahub.io"}, crd); err != nil {
```

Comment:
Just for sanity check? I see no references...

Reply:
Yes, it's a sanity check, following the scheme of other similar functions. I'm assuming it could be removed, since we try to get the `FeatureTracker` by name and ignore NotFound errors (assuming trying to fetch a non-existing crd returns a NotFound too)